src/app/view/components/main-legacy/main-legacy.component.html

```diff
@@ -1,6 +1,7 @@
-<dot-dialog></dot-dialog>
-<dot-contentlet-editor (close)="onCloseContentletEditor()"></dot-contentlet-editor>
+<!-- TODO: Move this to iframe-portlet-legacy -->
```

Comment:
I tried to do this, but the iframe module is all messed up and it was going to take me too much time.